extensions/cornerstone/src/commandsModule.js

```diff
@@ -348,6 +352,12 @@ const commandsModule = ({ servicesManager }) => {
       storeContexts: [],
       options: {},
     },
+    setCornerstoneLayout: {
+      commandFn: actions.setCornerstoneLayout,
+      storeContexts: ['viewports'],
+      options: {},
+      context: 'VIEWER',
+    },
```

Comment:
This doesn't appear to need/use `storeContexts`

Reply:
Removed. 👍